src/parser.h

```diff
@@ -49,6 +49,9 @@ typedef struct yp_lex_mode {
   // quotes).
   bool interp;
 
+  // Whether or not a YP_CONTEXT_DEF can use arguement forwarding
+  bool forwarding;
```

Comment:
I imagine this might change after: https://github.com/Shopify/yarp/issues/152 but I don't hate storing this here for now.